webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -0,0 +1,171 @@
+<div class="panel">
+    <header class="title">
+        <h2>Operating System</h2>
+    </header>
+    <div class="content">
+        <table class="tablesorter data-table">
+            <thead>
+                <tr>
+                    <th class="header">Rank</th>
+                    <th class="header">Operating System</th>
+                    <th class="header">Count</th>
+                    <th class="header">%</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for platform in query.facets.platform %}
+                <tr>
+                    <td>{{ loop.index }}</td>
+                    <td>{{ platform.term }}</td>
+                    <td>{{ platform.count }}</td>
+                    <td>{{ (platform.count / query.total * 100.0) | round(2) }}</td>
```

Comment:
If count is 3 and total is 100 that should yield 3.0% right?
```
>>> 3 / 10 * 100.0
0.0
```
Perhaps these numbers aren't `int`s but that's not easy to see. 
A simple solution would be to change to 
```
<td>{{ (100.0 * platform.count / query.total) | round(2) }}</td>
```